### Files look OK. In CLHS this topis is *COMPILE-FILE-PATHNAME*
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:variables:star-compile-file-pathname-star.txt	2017-04-27 21:40:35.000000000 +0100
+++ saved-pages-clus/cl:variables:star-compile-file-pathname-star.txt	2017-04-27 21:40:35.000000000 +0100
@@ -1,18 +1,38 @@
 Variable *COMPILE-FILE-PATHNAME*, *COMPILE-FILE-TRUENAME* 
  
 Value Type
  
 The value of *compile-file-pathname* must always be a pathname or nil. The value of *compile-file-truename* must always be a physical pathname or nil. 
 Initial Value
  
 nil. 
 Description
  
 During a call to compile-file, *compile-file-pathname* is bound to the pathname denoted by the first argument to compile-file, merged against the defaults; that is, it is bound to (pathname (merge-pathnames input-file)). During the same time interval, *compile-file-truename* is bound to the truename of the file being compiled. 
 At other times, the value of these variables is nil. 
 If a break loop is entered while compile-file is ongoing, it is implementation-dependent whether these variables retain the values they had just prior to entering the break loop or whether they are bound to nil. 
 The consequences are unspecified if an attempt is made to assign or bind either of these variables. 
-Examples None.
+Examples
+None.
  
 Affected By
  
@@ -16,8 +36,21 @@
  
 Affected By
  
 The file system. 
 See Also
  
-compile-file 
-Notes None.
\ No newline at end of file
### Possible formating problem.
+ Function COMPILE-FILE
+Notes
+None.
+\
\ No newline at end of file
